js/ad_manager_vast.js

```diff
@@ -867,8 +877,18 @@ OO.Ads.manager(function(_, $) {
      * @param {object} adWrapper The current Ad's metadata
      */
     this.playAd = function(adWrapper) {
-      currentAd = adWrapper;
+      if (adWrapper) {
+        currentAd = adWrapper;
+        if (currentAd.isAdRequest) {
+          loadAd(currentAd);
+        } else {
+          playLoadedAd(adWrapper);
+        }
+      }
+    }
 
+    var playLoadedAd = _.bind(function(adWrapper) {
```

Comment:
The vast ad manager uses underscore notation for private members. Can we change this to _playLoadedAd. Also, can we add a jsdoc for this?